corehq/apps/userreports/indicators/factory.py

```diff
@@ -156,6 +157,6 @@ def validate_spec(self, spec):
             raise BadSpecError(
                 _('Illegal indicator type: "{0}", must be one of the following choice: ({1})'.format(
                     spec['type'],
-                    ', '.join(self.constructor_map.keys())
+                    ', '.join(list(self.constructor_map.keys()))
```

Comment:
Assuming constructor_map is a dictionary, I think that you could just remove `.keys()` here. 

python2:

```
>>> ', '.join({'key1': 'value1', 'key2': 'value2'})
'key2, key1'
```
py3
```
>>> ', '.join({'key1': 'value1', 'key2': 'value2'})
'key1, key2'
```
@millerdev mentioned this to me when we first started thinking about this

Reply:
Not sure why ```python-modernize corehq/apps/userreports/ -w --future-unicode``` picked up on these changes.  Agree ````.keys()``` should be removed.  Is there a script setting to make that change, or do we have to do this one manually?